src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -33,6 +33,9 @@ const styles = {
   statusDetailCell: {
     align: 'center',
     style: { padding: '0.5rem' }
+  },
+  multiLineCellText: {
+    fontSize: 12
```

Comment:
This doesn't seem very specifically multi-line? I'm admittedly very css-naive but isn't it better just to say `fontSize: 12` inline whenever we need to?

Reply:
My rational here was that we are using a smaller font size for elements that are expected to render over multiple lines. I want the font size for date and submission ID to be the same (I wouldn't want 12 for one and 13 for the other, for example), so I defined a variable.